### DISCUSSION TITLE
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested

### DISCUSSION THREAD

**Discussion by mrtargaryen** [FreshRSS/FreshRSS#5078] [Q&A] [answered]
After following the Docker installation guide on a Raspberry Pi 4 I receive the warning message:

`WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested`

How do I resolve this?

- Raspberry Pi 4
- Raspbian Lite
- Docker 23

**Comment by Alkarex** [comment, marked as answer]
Hello,
For ARM-based platforms, please try `freshrss/freshrss:arm`, which is for `arm32v7`
https://github.com/FreshRSS/FreshRSS/blob/edge/Docker/README.md#our-docker-image-variants

Since we are not offering pre-builds for many platform, we have an easy way to build locally, which should address most of the cases, including `arm64v8`:
https://github.com/FreshRSS/FreshRSS/blob/edge/Docker/README.md#build-custom-docker-image
(You can even put that in a docker-compose if you like - I can provide an example if needed)

As to why we do not have builds for more platforms, in addition to resource consumption, see this duplicate:
https://github.com/FreshRSS/FreshRSS/issues/4089#issuecomment-1331124615

**Reply by mrtargaryen** [reply]
> Hello, For ARM-based platforms, please try `freshrss/freshrss:arm`, which is for `arm32v7` https://github.com/FreshRSS/FreshRSS/blob/edge/Docker/README.md#our-docker-image-variants
> 

I have a Raspberry Pi 3B and 4B, so I used `freshrss/freshrss:arm` on the 3B, and it worked, problem solved. Thank you.

**Reply by Alkarex** [reply]
I have documented an easy way to automatically build the Docker images locally from git:
https://github.com/FreshRSS/FreshRSS/pull/5237


```yml
    ...
    image: freshrss/freshrss:latest
    build:
      context: https://github.com/FreshRSS/FreshRSS.git#latest
      dockerfile: Docker/Dockerfile-Alpine
    ...
```
